src/Trakx.Kaiko.ApiClient.Stream.Tests/Integration/EnabledServices.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Trakx.Kaiko.ApiClient.Stream.Tests;
+
+internal static class EnabledServices
+{
+    internal const bool SpotExchangeRate = false;
```

Comment:
Should these settings be static in the code? If this must be changed, it will be needed to release a new version of this lib.